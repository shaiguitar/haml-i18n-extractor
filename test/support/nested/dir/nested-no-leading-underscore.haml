.container
  -# Breadcrumbs
  %ol.breadcrumb
    %li= link_to current_account.title, current_account
    %li
      = link_to account_settings_path do
        %i.fa.fa-fw.fa-cog
        %span.text Account Settings
    %li
      = link_to tariffs_path do
        %i.fa.fa-fw.fa-calculator
        %span.text Tariffs
    %li
      = link_to tariff_path(@tariff) do
        %i.fa.fa-fw.fa-calculator
        %span.text= @tariff.friendly_title
    %li.active
      %i.fa.fa-fw.fa-plus
      %span.text Add Volume Charge

  #pageTitle
    .pull-left
      %h1= current_account.title

    %ul.nav
      %li
        .btn-group
          = link_to new_tariff_path, class: 'btn btn-default btn-sm' do
            %i.fa.fa-fw.fa-plus
            %span.text New Tariff
      %li
        .btn-group
          = link_to account_settings_path, class: 'btn btn-default btn-sm' do
            %i.fa.fa-fw.fa-cog
            %span.text Settings

  .row
    .col-md-3
      = render partial: 'account_settings/navigation', locals: { parent: 'tariff' }

    .col-md-9
      = render 'form'
