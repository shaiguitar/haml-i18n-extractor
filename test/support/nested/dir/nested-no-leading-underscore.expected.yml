---
en:
  nested:
    dir:
      nested-no-leading-underscore:
        account_settings: Account Settings
        tariffs: Tariffs
        add_volume_charge: Add Volume Charge
        new_tariff: New Tariff
        settings: Settings
        account_settingsnavigation: account_settings/navigation
        tariff: tariff
        form: form
